src/honey_badger/message.rs

```diff
@@ -1,16 +1,16 @@
 // `threshold_sign::Message` triggers this Clippy lint, but `Box<T>` doesn't implement `Rand`.
 #![allow(clippy::large_enum_variant)]
 
-use rand::Rand;
-use rand_derive::Rand;
+use rand::distributions::{Distribution, Standard};
```

Comment:
Same question as above, should we be more explicit without distributions, i.e. preferring `Uniform` over `Standard`, because the former is definitely correct for choosing values, while the latter is not very descriptive?